_includes/sponsor-template.html

```diff
@@ -11,41 +11,43 @@ <h1 class="gala-heading">{{sponsor_type.sponsor-heading}}</h1>
     <!-- sponsors by level, such as 白金赞助 -->
     {% for sponsor_level in sponsor_type.sponsor-levels %}
     <div class="row">
-        <div class="col home-section-header pb-small">
+        <div class="col home-section-header pt-9" >
             <!-- sponsor level heading, such as the text 白金赞助 or 虚位以待 -->
-            <h2 class="gala-heading underline">{{sponsor_level.level-heading}}</h2>
-            <div class="home-section-content">
-                <p class="py-2"><br></p>
+            <div class="row">
+                <div class="col pb-3 text-center">
+                    <h1 class="underline">{{sponsor_level.level-heading}}</h1>
+                </div>
             </div>
         </div>
     </div>
```

Comment:
Move this block of code under `{% if sponsor_level.sponsor-elements %}`. If this level does not contain any sponsor, why bother show the title?